tests/callbacks/test_tqdm_progress_bar.py

```diff
@@ -347,10 +348,10 @@ def test_tqdm_progress_bar_value_on_colab(tmpdir):
         [2, 3, 1, [1, 2, 3, 4, 5], [1, 2, 3]],
         [0, 0, 3, None, None],
         [1, 0, 3, [1], None],
-        [1, 1, 3, [1, 2], [1]],
+        [1, 1, 3, [2], [1]],
         [5, 0, 3, [3, 5], None],
-        [5, 2, 3, [3, 5, 7], [2]],
-        [5, 2, 6, [5, 7], [2]],
```

Comment:
Same

Reply:
same here. `train batches = 5` and `val batches = 2` thus `total = 7` and refresh_rate=3, so for main progress bar updates should happen with multiples of 3 i.e 3, 6, 7(this is the end)